website/addons/box/model.py

```diff
@@ -86,21 +86,9 @@ def api(self):
     def display_name(self):
         return '{0}: {1}'.format(self.config.full_name, self.folder_id)
 
-    @property
-    def has_auth(self):
-        """Whether an access token is associated with this node."""
-        return bool(self.user_settings and self.user_settings.has_auth)
-
-    @property
-    def complete(self):
-        return bool(self.has_auth and self.user_settings.verify_oauth_access(
-            node=self.owner,
-            external_account=self.external_account,
-        ))
-
     def fetch_folder_name(self):
         self._update_folder_data()
-        return self.folder_name.replace('All Files', '/ (Full Box)')
+        return (getattr(self, 'folder_name') or '').replace('All Files', '/ (Full Box)')
```

Comment:
This is super minor, but you could also specify a default value using `getattr(self, 'folder_name', '')`